src/main/flight/position.c

```diff
@@ -211,8 +211,7 @@ void calculateEstimatedAltitude(void)
     DEBUG_SET(DEBUG_ALTITUDE, 3, estimatedVario);
 #endif
     DEBUG_SET(DEBUG_RTH, 1, lrintf(displayAltitudeCm / 10.0f));
-    DEBUG_SET(DEBUG_ALTHOLD, 1, lrintf(zeroedAltitudeCm));
-    DEBUG_SET(DEBUG_GPS_RESCUE_THROTTLE_PID, 1, lrintf(zeroedAltitudeCm));
+    DEBUG_SET(DEBUG_AUTO_PILOT_ALTITUDE, 2, lrintf(zeroedAltitudeCm));
```

Comment:
This is better suited in `altitudeControl()` where the other autopilot debugs exist. I made a suggestion there as well. 
(Similar for the RTH debug but it's not part of your PR)
```suggestion
```

Reply:
I agree.  However, the nice thing about doing it here is that the debug then shows the altitude value all the time, not only when the function to control it is running.  I found that useful.  And, if we move it to the client code, we have to get the value into the client code, purely for the debug, but we don't need it there.
Hence I think it's best where it is.